config/search_scheme.json

```diff
@@ -71,60 +96,117 @@
     }
   },
   "mappings": {
+    "_meta": {
+        "description": "Created by john mcbride for testing"
```

Comment:
Are such personalised descriptions allowed?

Reply:
Good spot - I'll update the description to say "Mappings for alphabetical search"